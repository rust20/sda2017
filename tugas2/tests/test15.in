15 6 8 500 110 110
Arturia Pendragon;1;9999;panah
Amakusha;2;9999;panah
Nazuna;3;9999;panah
Feget;4;9999;panah
Emiye;25;9999;panah
Queen Riana;0;9999;panah
Raven;45;30000;panah
Rigel;40;30000;panah
Stella;20;9999;panah
Jyanu;22;9999;panah
Hector;10;9999;panah
Hercules;20;9999;panah
Gilgamesh;12;9999;panah
Cu;14;9999;pedang
Kizuna;20;9999;pedang
1;110;Gilgamesh,Hercules,Rigel
3;2;Arturia Pendragon,Amakusha
1;3;Nazuna,Emiye,Jyanu,Hector,Queen Riana,Stella
1;5;Raven
109;110;Cu,Feget
107;1;Kizuna
1;107;11;1;pedang;3
109;2;215;1;pedang;3
2;1;40;13;pedang;4
110;1;201;300;pedang;1
110;110;10000;200;pedang;1000
63;54;10000;300;pedang;400
49;59;900;1200;pedang;20
2;110;77;34;pedang;20
..S#S.......................................#............................................................#D..S
D.##...........................................#.........................................................#...D
#S##.....................................................................................................#....
#.#.................................#....................................................................#....
#.#.......................................#.........................................#....................#....
#.#......................................................................................................#....
.........................................................................................................#....
...................................................................#.....................................#....
.........................................................................................................#....
.................#.......................................................................................#....
.........................................................................................................#....
.........................................................................................................#...#
..............................................................................................................
..............................................................................................................
.............................................................#................................................
..........................................#...................................................................
..............................................................................................................
.......................#......................................................................................
............................................................#........................#........................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
.....................................#........................................................................
.....................................#M#..................D...................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
.....................................................D........................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
S.............................................................................................................
..............................................................................................................
.D...........................................................................................................S
D............................................................................................................D
