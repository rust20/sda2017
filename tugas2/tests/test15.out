1,107 RUN, kekuatan maksimal sejumlah: 0
1,110 Pahlawan yang ikut:Rigel,Gilgamesh,Hercules
2,110 BATTLE, kekuatan: 99996, pahlawan: Rigel,Gilgamesh,Hercules
109,110 Pahlawan yang ikut:Feget,Cu
110,110 BATTLE, kekuatan: 129993, pahlawan: Rigel,Gilgamesh,Hercules,Feget,Cu
49,59 BATTLE, kekuatan: 129993, pahlawan: Rigel,Gilgamesh,Hercules,Feget,Cu
63,54 BATTLE, kekuatan: 129993, pahlawan: Rigel,Gilgamesh,Hercules,Feget,Cu
1,5 Pahlawan yang ikut:Raven
3,2 Pahlawan yang ikut:Arturia Pendragon,Amakusha
1,3 Pahlawan yang ikut:Queen Riana,Nazuna,Hector,Stella,Jyanu,Emiye
2,1 BATTLE, kekuatan: 159996, pahlawan: Rigel,Raven,Gilgamesh,Hercules
109,2 BATTLE, kekuatan: 139998, pahlawan: Rigel,Raven,Gilgamesh
107,1 Pahlawan yang ikut:Kizuna
110,1 BATTLE, kekuatan: 60000, pahlawan: Rigel
Akhir petualangan Gudako
Level Gudako: 8958
Level pahlawan:
Rigel: 2049
Gilgamesh: 1749
Hercules: 1748
Feget: 1701
Cu: 1701
Raven: 15
Arturia Pendragon: 1
Amakusha: 1
Queen Riana: 1
Nazuna: 1
Hector: 1
Stella: 1
Jyanu: 1
Emiye: 1
Kizuna: 1
